layers/+completion/spacemacs-helm/config.el

```diff
@@ -0,0 +1,14 @@
+;;; config.el --- Spacemacs-helm Layer config File
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defvar spacemacs-helm-basic-config nil
+  "If non nil, don't bind keys globally in spacemacs-helm and
+don't enable helm-mode.")
```

Comment:
I'm not for this variable, it does not make sense, Spacemacs should always configure Helm whenever it is used by a package. The issue is that with the current layer system we cannot say "configure this package because it is a dependency of another package".

But we can hack around by adding a local package called `basic-helm-config` to `spacemacs` layer. This local package would have no file in local directory.

I guess we will have to do the same for Ivy but we can keep this for another PR, this one being rather big already.